packages/integrations/src/env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="astro/client" />
```

Comment:
can you remove this file from the commit?

Reply:
Sure thing! Sorry for the small details; I'm new to Astro, and I wasn't sure if this was needed.